### PR TITLE
Update SignUpVCMethods.swift

### PR DESCRIPTION
**Pull request by baquer**
check the SignUpVCMethods

### CODE REVIEW

File: Susi/Controllers/SignUpController/SignUpVCMethods.swift [hunk 1]
```diff
@@ -129,7 +129,7 @@ extension SignUpViewController {
                 emailTextField.dividerActiveColor = .green
             }
         } else if textField == passwordTextField, let password = passwordTextField.text {
-            if password.isEmpty || password.characters.count < 5 {
+            if password.isEmpty && password.characters.count < 5 {
```
Comment (chashmeetsingh) [inline Susi/Controllers/SignUpController/SignUpVCMethods.swift]: Hi,
Why replace `||` with `&&`?

**Comment by baquer** [comment]
Using && if both our condition gets true then we can proceed further 
While using or || if one of the condition gets true we can proceed but it's not much secure and password strength is also not good
We can also strength the password by applying condition that the first letter of our password should be uppercase and its contain one special character also

File: Susi/Controllers/SignUpController/SignUpVCMethods.swift (hunk 1 above)
Comment (baquer) [reply Susi/Controllers/SignUpController/SignUpVCMethods.swift]: Using && if both our condition gets true then we can proceed further
While using or || if one of the condition gets true we can proceed but it's not much secure and password strength is also not good
We can also strength the password by applying condition that the first letter of our password should be uppercase and its contain one special character also

Comment (chashmeetsingh) [reply Susi/Controllers/SignUpController/SignUpVCMethods.swift]: Here, the statement can be reduced to just checking the character length.
`if password.characters.count < 5` right?

Comment (baquer) [reply Susi/Controllers/SignUpController/SignUpVCMethods.swift]: Yes right.

Comment (chashmeetsingh) [reply Susi/Controllers/SignUpController/SignUpVCMethods.swift]: Please make changes and commit.

**Comment by chashmeetsingh** [comment]
We can skip and directly use the check to count the number of characters if they are less than 5. That would work too.

**Comment by chashmeetsingh** [comment]
Closing this due to inactivity

**Pull request closed without merging**